### PR TITLE
[DRUP-430] terms conditions

### PR DESCRIPTION
**Pull request by Jaesin**

### CODE REVIEW

**Review by minnur — APPROVED** [review]
Looks good.

**Comment by cnovak** [comment]
@minnur and @Jaesin confirmed signed CLA

File: src/Entity/SubscriptionInterface.php [hunk 1]
```diff
@@ -69,4 +70,12 @@ public function isSubscriptionActive(): bool;
    */
   public function getSubscriptionStatus(): string;
 
+  /**
+   * Get's data for the `apigee_tnc` field formatter.
```
Comment (cnovak) [inline src/Entity/SubscriptionInterface.php]:
```suggestion
   * Gets data for the `apigee_tnc` field formatter.
```

File: src/Monetization.php [hunk 2]
```diff
@@ -196,6 +198,72 @@ public function getCompanyPrepaidBalances(CompanyInterface $company, \DateTimeIm
...
+  /**
+   * {@inheritdoc}
+   */
+  public function acceptLatestTermsAndConditions(string $developer_id): ?LegalEntityTermsAndConditionsHistoryItem {
+    try {
+      return $this->sdk_controller_factory->developerTermsAndConditionsController($developer_id)
+        ->acceptTermsAndConditionsById($this->getLatestTermsAndConditions()->id());
+    }
+    catch (\Throwable $t) {
+      // @TODO: Log errors ?
```
Comment (cnovak) [inline src/Monetization.php:255]: We shouldn't just swallow exceptions, need ticket to work on this later

**Review by cnovak — APPROVED** [review]
Will approve but please add ticket to backlog for handling exceptions

**Comment by Jaesin** [comment]
I'm seeing two copies of the accept form. 
image

File: src/Monetization.php (hunk 2 above)
Comment (Jaesin) [reply src/Monetization.php:255]: I created an issue.

**Comment by Jaesin** [comment]
The double form issue was because of an issue with the rebase.

**Pull request merged**